test/unittest/replica-7f4d4895ff58.lua

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env tarantool
+
+box.cfg {
+    listen = os.getenv('LISTEN'),
+    replication = os.getenv('MASTER'),
+    memtx_memory = 107374182
```

Comment:
Default 256MiB should be okay. Don't clog box.cfg with options you don't need (here and in other instance files).

Reply:
Fixed.